orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/temporal/BeforeOrAt.feature

```diff
@@ -8,18 +8,18 @@ Feature: User can specify that a datetime date is lower than, or the same as, a
   Scenario: Running beforeOrAt request that includes datetime field with date (YYYY-MM-DD) values that has invalid date should fail
     Given foo is before or at 2019-15-32T00:00:00.000Z
     And the generator can generate at most 5 rows
-    Then the profile is invalid because "Date string '2019-15-32T00:00:00.000Z' must be in ISO-8601 format: Either yyyy-MM-ddTHH:mm:ss.SSS\[Z\] between 0001-01-01T00:00:00.000Z and 9999-12-31T23:59:59.999Z or yyyy-mm-dd between 0001-01-01 and 9999-12-31"
+    Then the profile is invalid with error containing "must be in ISO-8601 format"
     And no data is created
 
   Scenario: Running beforeOrAt request that includes datetime field with date and time (YYYY-MM-DDTHH:MM:SS) values that has invalid time should fail
     Given foo is before or at 2018-10-01T25:25:05.000Z
     And the generator can generate at most 5 rows
-    Then the profile is invalid because "Date string '2018-10-01T25:25:05.000Z' must be in ISO-8601 format: Either yyyy-MM-ddTHH:mm:ss.SSS\[Z\] between 0001-01-01T00:00:00.000Z and 9999-12-31T23:59:59.999Z or yyyy-mm-dd between 0001-01-01 and 9999-12-31"
+    Then the profile is invalid with error containing "must be in ISO-8601 format"
     And no data is created
 
   Scenario: Running beforeOrAt request that includes datetime field with date and time (YYYY-MM-DDTHH:MM:SS) values that has invalid year should fail
     Given foo is before or at 0000-01-10T00:00:00.000Z
-    Then the profile is invalid because "Dates must be between 0001-01-01T00:00:00.000Z and 9999-12-31T23:59:59.999Z"
+    Then the profile is invalid with error containing "Dates must be between 0001-01-01T00:00:00.000Z and 9999-12-31T23:59:59.999Z | Field: foo | Constraint: beforeOrAt | Rule: Unnamed rule"
     And no data is created
```

Comment:
Is there a reason this check is more specific than the others in this fiel (`Then the profile is invalid with error containing "must be in ISO-8601 format"`)

Reply:
Not that I am aware of